src/nunit.gui.core/Model/TestModel.cs

```diff
@@ -202,12 +205,12 @@ public void NewProject()
 
         public void NewProject(string filename)
         {
-            MessageBox.Show("It's not yet decided if we will implement saving of projects. The alternative is to require use of the project editor, which already supports this.", "Not Yet Implemented");
+            _messageDisplay.Info("It's not yet decided if we will implement saving of projects. The alternative is to require use of the project editor, which already supports this.", "Not Yet Implemented");
```

Comment:
Rather than issuing a message here, we should call _view.MessageDisplay in the presenter where we call this.